src/tests/ipfixlolib/example_code_2.c

```diff
@@ -41,7 +41,7 @@ struct config myconfig = {
     .coll_ip4_addr = DEFAULT_COLLECTOR_IP_ADDRESS,
     .coll_port = 0,
     .transport_protocol = UDP,
-    .mtu = 0,
+    .mtu = 576,
```

Comment:
Can you add a comment on why you chose this MTU value?

Reply:
576 is the minimum datagram size for IPV4.
I'll change it to 1492, the maximum for Ethernet LLC, and and a comment.